geode-core/src/test/java/org/apache/geode/internal/cache/AbstractRegionEntryTest.java

```diff
@@ -80,6 +80,7 @@ public void whenPrepareValueForCacheCalledWithOffHeapEntryHasNewCachedSerialized
     Object o = DataSerializer.readObject(dataInputStream);
     assertNotNull(entryEvent.getCachedSerializedNewValue());
     assertEquals(o, value);
+    MemoryAllocatorImpl.freeOffHeapMemory();
```

Comment:
put this call to freeOffHeapMemory in a finally block whose "try" is right after you called "createForUnitTest".

Reply:
Yes, that's correct.